site/src/lib/nextCache.tsx

```diff
@@ -0,0 +1,39 @@
+import path from 'path';
+import fs from 'fs';
+
+const cacheDir = path.join(process.cwd(), '.next/cache');
+const cachePath = (cacheKey: string) => path.join(cacheDir, `${cacheKey}.json`);
+
+const read = (cacheKey: string) => {
+  if (fs.existsSync(cachePath(cacheKey))) {
+    const iconCache = fs.readFileSync(cachePath(cacheKey), "utf8")
+    return JSON.parse(iconCache)
+  }
+
+  return null
+}
+
+const write = (cacheKey: string, content: any) => {
+  if (!fs.existsSync(cacheDir)) {
+    fs.mkdirSync(cacheDir)
+  }
+
+  fs.writeFileSync(cachePath(cacheKey), JSON.stringify(content), 'utf-8')
```

Comment:
We can't use `fs.write` during runtime in Vercel.
What we can do is create a cache on build time. 
So we need to create a separate script we execute when the build command is fired.